### PR TITLE
Add changelog and screenshots for fastlane

### PR DESCRIPTION
**Pull request by koiuo**

### CODE REVIEW

File: fastlane/metadata/android/en-US/changelogs/100000.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+First official release under a new name after forking [Hayai Launcher](https://github.com/edgarvperes/HayaiLauncher).
```
Comment (koiuo) [inline fastlane/metadata/android/en-US/changelogs/100000.txt]: Markdown doesn't seem to be supported, however for simplicity I decided to leave markdown here as is

If html tags are supported (like in full_description.txt), then we could render markdown to html.

@IzzySoft , do you know if html tags in changelog entries are allowed? Thanks!

**Comment by IzzySoft** [comment]
Markdown links won't work (except with my repo that is, and even there only in `full_description.txt` if defined as Markdown in the app's metadata – I use some "extensions" on top of what F-Droid supplies). Details on what's possible [here in my wiki](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Descriptions). For most widest compatibility, I'd suggest what I call "markdown lite" – again, for `full_description.txt`.

Changelogs are AFAIK plain text only. So the above would be shown as … above, just without the highlighting. Try it once and see – let me know if I was wrong, please :wink:

**Comment by koiuo** [comment]
@avuton , I'm merging this.
I apologize for my boldness, but I have spare minute right now and would like to make fastlane work. Please let me know if you have concerns regarding this PR and I'll make sure rollback and cleanup everything.

**Comment by avuton** [comment]
I didn't realize I was being waited upon, you're trusted, merge as necessary for releases.

**Pull request merged**